src/main/java/org/sonar/squidbridge/annotations/AnnotationBasedRulesDefinition.java

```diff
@@ -63,8 +70,8 @@
  * /org/sonar/l10n/[languageKey]/rules/[repositoryKey]/ruleKey.html</li>
  * </ul>
  *
- * @since 2.5
  */
+@Deprecated
 public class AnnotationBasedRulesDefinition {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Do not forget to remove this deprecated code someday. ([squid:S1133](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sslr-squid-bridge%3Asslr-squid-bridge&branch=SSLRSQBR-71&issues=AWHXxlpEaf12pdu-6jE7&open=AWHXxlpEaf12pdu-6jE7)

[ISSUE_KEY]: AWHXxlpEaf12pdu-6jE7